lib/plugins/audit.js

```diff
@@ -25,6 +25,10 @@ var HttpError = require('restify-errors').HttpError;
 function auditLogger(options) {
```

Comment:
Since this class is no longer just writing logs, change its name to `audit`  instead.